examples/worklets/bitcrusher.js

```diff
@@ -0,0 +1,77 @@
+class Bitcrusher extends AudioWorkletProcessor {
+    static get parameterDescriptors() {
+        return [{
+            name: 'bitDepth',
+            defaultValue: 12,
+            minValue: 1,
+            maxValue: 16
+        }, {
+            name: 'frequencyReduction',
+            defaultValue: 0.5,
+            minValue: 0,
+            maxValue: 1
+        }];
+    }
+
+    constructor(options) {
+      process.stdout.write(`++ in constructor: ${JSON.stringify(options, null, 2)}\n`);
```

Comment:
Don't really get why these prints are swallowed, is this a regression ?

Reply:
Yeah this used to work (via console.log even) when we started using `setImmediate` in the worker loop

Reply:
The logs just re-appeared on my side, even with `console.log`, weird...